core/communication/src/main/java/com/cognifide/aet/queues/DefaultJmsConnection.java

```diff
@@ -17,35 +17,25 @@
 
 import com.cognifide.aet.communication.api.queues.JmsConnection;
 import com.cognifide.aet.communication.api.queues.JmsEndpointConfig;
-import java.util.Map;
+import com.cognifide.aet.queues.configuration.DefaultJmsConnectionConf;
 import javax.jms.Connection;
 import javax.jms.JMSException;
 import javax.jms.Session;
 import org.apache.activemq.ActiveMQConnectionFactory;
-import org.apache.felix.scr.annotations.Activate;
-import org.apache.felix.scr.annotations.Component;
-import org.apache.felix.scr.annotations.Deactivate;
-import org.apache.felix.scr.annotations.Property;
-import org.apache.felix.scr.annotations.Reference;
-import org.apache.felix.scr.annotations.Service;
 import org.osgi.service.cm.ConfigurationAdmin;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@Service
-@Component(immediate = true, metatype = true, description = "AET JMS Connection", label = "AET Default JMS Connection")
+@Component(service = JmsConnection.class, immediate = true)
```

Comment:
`service = JmsConnection.class` is redundant - by default the service type equals the type of implemented interface